queue/lib/txgh-queue/job.rb

```diff
@@ -57,7 +59,15 @@ def handle_transifex_hook(project, repo, payload)
         logger: logger
       )
 
-      handler.execute
+      execute(handler)
+    end
+
+    def execute(handler)
+      if TxghQueue::Config.processing_enabled?
+        handler.execute
+      else
+        respond_with(200, 'Ok')
```

Comment:
I think it just wants something in the 2 or 300s. Is there a more appropriate response code?